### ISSUE TITLE
OH2 > OH3 update: frontail web is not updated

### ISSUE THREAD

**Issue by eikowagenknecht** [openhab/openhabian#1232]
Seems like frontail web is not updated and thus still showing the OH2 logs.

The paths in `sudo nano /etc/systemd/system/frontail.service` need to be changed.

```
After=openhab.service
...
xecStart=/usr/lib/node_modules/frontail/bin/frontail --ui-highlight --ui-highlight-preset /usr/lib/node_modules/frontail/preset/openhab.json -t openhab -l 2000 -n 200 /var/log/openhab/openhab.log /var/log/openhab/events.log
```

instead of

```
After=openhab2.service
...
xecStart=/usr/lib/node_modules/frontail/bin/frontail --ui-highlight --ui-highlight-preset /usr/lib/node_modules/frontail/preset/openhab.json -t openhab -l 2000 -n 200 /var/log/openhab2/openhab.log /var/log/openhab2/events.log
```

Also edit `/usr/lib/node_modules/frontail/preset/openhab.json` and change the folders there for the syntax highlighting. "openhab.css" also needs to be changed to not display "openHAB 2" any more, though I'm not sure where to find that file.

**Comment by eikowagenknecht** [comment]
Part of this seems to be related to #1230, but even then openhab.json and openhab.css seem not to be changed, only frontail.service is.

**Comment by mstormi** [comment]
Fixed in [b976a6dd6d8d5056cfb0dda4f7e42269ab968a4e](https://github.com/openhab/openhabian/commit/b976a6dd6d8d5056cfb0dda4f7e42269ab968a4e)
